openshift/qontract-reconcile.yaml

```diff
@@ -37,11 +37,11 @@ objects:
             value: ${SLEEP_DURATION_SECS}
           resources:
             limits:
-              cpu: 50m
-              memory: 400Mi
+              cpu: ${QR_AWS_GARBAGE_COLLECTOR_CPU_LIMIT}
```

Comment:
Those numbers are coming from helm... https://github.com/app-sre/qontract-reconcile/blob/master/helm/qontract-reconcile/values.yaml

Reply:
/facepalm
closing this then...